### PR TITLE
[DO NOT MERGE] Brings back Freon.

### PR DESCRIPTION
**Pull request by SireTurret**
## Changelog
:cl: Turret
add: Added back freon, a really cold gas which is really good if you wanna contain the supermatter quickly.
/:cl:

## About The Pull Request

THIS SHOULDN'T BE MERGED YET, HALF THE SHIT ISN'T EVEN IN YET.

## Why It's Good For The Game

Because the removal of freon was pretty retarded and unnecessary.

### CODE REVIEW

**Comment by SireTurret** [comment]
Also made the hypernob and freon different as a sprite.

**Comment by notoriousBIGGIOS** [comment]
yes

File: code/modules/atmospherics/gasmixtures/gas_types.dm [hunk 1]
```diff
@@ -85,12 +85,21 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	fusion_power = 8
 	rarity = 500
 
+/datum/gas/freon
+	id = "freon"
+	specific_heat = 2000
+	name = "Freon"
+	gas_overlay = "freon"
```
Comment (ghost) [inline code/modules/atmospherics/gasmixtures/gas_types.dm:92]: should be in `hippiestation/code/modules/atmospherics/gasmixtures/gas_types.dm`

**Comment by ghost** [comment]
based and brappilled

**Comment by SireTurret** [comment]
im gonna make a new pr

**Pull request closed without merging**